### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by alansouzati**

### CODE REVIEW

File: src/js/components/DropButton/__tests__/DropButton-test.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    const component = mount(
+      <DropButton control={<input />} onClose={onClose}>
+        <div id='drop-contents'>drop contents</div>
+      </DropButton>, {
+        attachTo: document.body.firstChild,
+      }
+    );
+
+    expect(component.getDOMNode()).toMatchSnapshot();
+    expect(document.getElementById('drop-contents')).toBeNull();
```
Comment (ericsoderberghp) [inline src/js/components/DropButton/__tests__/DropButton-test.js]: Should be expectPortal

File: src/js/components/Grommet/__tests__/Grommet-test.js [hunk 2]
```diff
@@ -21,3 +38,18 @@ test('Grommet hpe theme renders', () => {
...
+  );
+  expect(component.getDOMNode()).toMatchSnapshot();
+
+  expect(document.body.querySelector('[aria-live]')).toMatchSnapshot();
+
+  setTimeout(() => {
+    // should clear the aria-live container
+    expect(document.body.querySelector('[aria-live]')).toMatchSnapshot();
+    done();
+  }, 600);
```
Comment (ericsoderberghp) [inline src/js/components/Grommet/__tests__/Grommet-test.js]: Why 600? Maybe a comment, especially if this has to align with a timeout inside Grommet.

File: src/js/components/Grommet/__tests__/Grommet-test.js [hunk 3]
```diff
@@ -21,3 +38,18 @@ test('Grommet hpe theme renders', () => {
...
   expect(tree).toMatchSnapshot();
 });
+
+test('Grommet announces', (done) => {
+  const component = mount(
+    <Grommet><TestAnnouncer /></Grommet>
+  );
+  expect(component.getDOMNode()).toMatchSnapshot();
+
+  expect(document.body.querySelector('[aria-live]')).toMatchSnapshot();
```
Comment (ericsoderberghp) [inline src/js/components/Grommet/__tests__/Grommet-test.js]: expectPortal

File: src/js/components/Layer/__tests__/Layer-test.js [hunk 4]
```diff
@@ -41,35 +42,71 @@ class FakeLayer extends Component {
...
+    expectPortal('left-test').toMatchSnapshot();
   });
 
-  test('Layer mounts', () => {
-    const component = mount(<FakeLayer />);
-    expect(component.getDOMNode()).toMatchSnapshot();
-    component.unmount();
+  test('aligns right', () => {
+    mount(
+      <Layer id='right-test' position='left'>
```
Comment (ericsoderberghp) [inline src/js/components/Layer/__tests__/Layer-test.js]: `position='right'`

File: src/js/components/Layer/__tests__/Layer-test.js [hunk 5]
```diff
@@ -41,35 +42,71 @@ class FakeLayer extends Component {
...
+        This is a layer
+      </Layer>
+    );
+    expectPortal('right-test').toMatchSnapshot();
+  });
+
+  test('aligns top', () => {
+    mount(
+      <Layer id='top-test' position='left'>
```
Comment (ericsoderberghp) [inline src/js/components/Layer/__tests__/Layer-test.js]: `position='top'`

File: src/js/components/Layer/__tests__/Layer-test.js [hunk 6]
```diff
@@ -41,35 +42,71 @@ class FakeLayer extends Component {
...
+        This is a layer
+      </Layer>
+    );
+    expectPortal('top-test').toMatchSnapshot();
+  });
+
+  test('aligns bottom', () => {
+    mount(
+      <Layer id='bottom-test' position='left'>
```
Comment (ericsoderberghp) [inline src/js/components/Layer/__tests__/Layer-test.js]: `position='bottom'`

File: src/js/components/DropButton/__tests__/DropButton-test.js (hunk 1 above)
Comment (alansouzati) [reply src/js/components/DropButton/__tests__/DropButton-test.js]: expect to be null we don't need to call expectPortal ;)

**Pull request merged**